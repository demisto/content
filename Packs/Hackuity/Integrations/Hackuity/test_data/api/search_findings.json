{
  "~~~NOTE~~~": "unknown data should be ignored",
  "count": 2,
  "kind": "hy#searchFindingList",
  "limit": 20,
  "offset": 0,
  "searchFindings": [
    {
      "~~~NOTE~~~": "unknown data should be ignored",
      "assetId": "DE9qZUZkOequ21mzpmTK69",
      "assetAbsoluteName": "notcorebusiness.notld",
      "assetType": "DOMAIN",
      "findingStatus": {
        "ignored": true,
        "score": {
          "environmental": 5.8,
          "hyScore": 671
        },
        "state": "OPEN",
        "subState": "POTENTIAL"
      },
      "findingId": "nDoKTtbR6tanVtgmtP5BQo",
      "vulnTypeId": "123"
    },
    {
      "assetAbsoluteName": "192.168.13.37",
      "assetId": "zjLoHvffL9c3i2hyVCF3h2",
      "assetType": "IPV4",
      "findingId": "RE5isckTYGbU9LVCc76NTV",
      "findingStatus": {
        "lastClosedAt": "2022-01-01T13:37:42Z",
        "score": {
          "environmental": 10,
          "hyScore": 980
        },
        "state": "CLOSED",
        "subState": "FP"
      },
      "findingAttributes": { "cve_id": "CVE-2014-0160" },
      "vulnTypeId": "456"
    }
  ],
  "total": 2
}
