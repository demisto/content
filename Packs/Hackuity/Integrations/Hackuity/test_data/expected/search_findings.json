[
  {
    "Asset": {
      "ID": "DE9qZUZkOequ21mzpmTK69",
      "Name": "notcorebusiness.notld",
      "Type": "DOMAIN"
    },
    "Attributes": {},
    "Score": { "CVSS": 5.8, "TRS": 671 },
    "ID": "nDoKTtbR6tanVtgmtP5BQo",
    "Status": { "Ignored": true, "State": "OPEN", "SubState": "POTENTIAL", "LastClosedAt": null },
    "VulnType": { "ID": "123", "Name": "Lorem ipsum" }
  },
  {
    "Asset": {
      "ID": "zjLoHvffL9c3i2hyVCF3h2",
      "Name": "192.168.13.37",
      "Type": "IPV4"
    },
    "Attributes": { "cve_id": "CVE-2014-0160" },
    "Score": { "CVSS": 10, "TRS": 980 },
    "ID": "RE5isckTYGbU9LVCc76NTV",
    "Status": { "Ignored": false, "State": "CLOSED", "SubState": "FP", "LastClosedAt": "2022-01-01T13:37:42Z" },
    "VulnType": { "ID": "456", "Name": "Dolor amet" }
  }
]
